Vadym_Bilestkyi/2/Task8.py

```diff
@@ -0,0 +1,14 @@
+def is_armstrong(number):
+    digits = [int(letter) for letter in str(number)]
```

Comment:
You will get an exception if letter is not int. Si it's better to use isdigit() method to validate numbers.